docs/site/todo-tutorial-geocoding-service.md

```diff
@@ -14,7 +14,7 @@ To call other APIs and web services from LoopBack applications, we recommend to
 use Service Proxies as a design pattern for encapsulating low-level
 implementation details of communication with 3rd-party services and providing
 JavaScript/TypeScript API that's easy to consume e.g. from Controllers. See
-[Calling other APIs and web services](./Calling-other-APIs-and-web-services.md)
+[Calling other APIs and web services](https://loopback.io/doc/en/lb4/Calling-other-APIs-and-web-services.html)
```

Comment:
@dhmlau It would be great to capture the information about `jekyll-relative-links` in our docs for people contributing documentation and evangelize this info amongst people doing pull request reviews.

Reply:
Actually, I think the broken link is due to the incorrect case in the file name.  The file name should be `Calling-other-APIs-and-Web-Services.md`.
https://github.com/strongloop/loopback-next/blob/master/docs/site/Calling-other-APIs-and-Web-Services.md

Reply:
Makes sense. It makes me wonder - is there any easy way how to lint our markdown files to ensure relative links are using correct letter case?

@nguyenjk could you please update your pull request and just fix the filename case, preserving the link as a relative path to `.md` file?